tests/conftest.py

```diff
@@ -37,8 +37,29 @@
     "s03": mock_paths,
     "i04": mock_paths,
     "s04": mock_paths,
+    "i24": mock_paths,
 }
 
+BANNED_PATHS = [Path("/dls"), Path("/dls_sw")]
```

Comment:
Can we just ban absolute paths altogether? /dev/null we can still get with `os.devnull`, otherwise everything should be included with the module

Reply:
I didn't want to exclude /tmp as there are lots of things that write to the tmp directory. As it's a patch on the `open` builtin I didn't want to be too overreaching in case some dependency of ours tries to do something reasonable.
I think this will catch 99% of things, I don't know what other directories we are likely to be touching ourselves.